ci/cloudbuild/builds/gcc-63.sh

```diff
@@ -28,7 +28,8 @@ cmake -GNinja -H. -Bcmake-out \
   -DGOOGLE_CLOUD_CPP_ENABLE="$(features::always_build_cmake)" \
   -DBUILD_SHARED_LIBS=yes -DGOOGLE_CLOUD_CPP_ENABLE_CCACHE=ON
 cmake --build cmake-out
+mapfile -t ctest_args < <(ctest::common_args)
 (
   cd cmake-out
-  ctest --output-on-failure -LE "integration-test" -j "$(nproc)"
+  ctest "${ctest_args[@]}" -LE "integration-test"
```

Comment:
Can we use `env -C cmake-out ctest ...`, instead of the sub-shell, to match the other files?

Reply:
Unfortunately we cannot, this runs on Ubuntu:16.04 where `env` does not support `-C`.  I will add a comment.